### ISSUE TITLE
The keyword 'package' is reserved

### ISSUE THREAD

**Issue by joshhornby** [jls/nightwatch-html-reporter#13]
I get the following error when running the basic test on the nightwatch home page

```
Reading reports directory...
/usr/local/lib/node_modules/nightwatch-html-reporter/node_modules/jade/node_modules/with/node_modules/acorn/dist/acorn.js:1023
  throw err;
        ^
SyntaxError: The keyword 'package' is reserved (160:10)
    at Parser.pp.raise (/usr/local/lib/node_modules/nightwatch-html-reporter/node_modules/jade/node_modules/with/node_modules/acorn/dist/acorn.js:1021:13)
    at Parser.pp.parseIdent (/usr/local/lib/node_modules/nightwatch-html-reporter/node_modules/jade/node_modules/with/node_modules/acorn/dist/acorn.js:732:240)
    at Parser.pp.parseBindingAtom (/usr/local/lib/node_modules/nightwatch-html-reporter/node_modules/jade/node_modules/with/node_modules/acorn/dist/acorn.js:1136:19)
    at Parser.pp.parseVar (/usr/local/lib/node_modules/nightwatch-html-reporter/node_modules/jade/node_modules/with/node_modules/acorn/dist/acorn.js:1987:20)
    at Parser.pp.parseVarStatement (/usr/local/lib/node_modules/nightwatch-html-reporter/node_modules/jade/node_modules/with/node_modules/acorn/dist/acorn.js:1883:8)
    at Parser.pp.parseStatement (/usr/local/lib/node_modules/nightwatch-html-reporter/node_modules/jade/node_modules/with/node_modules/acorn/dist/acorn.js:1683:19)
    at Parser.pp.parseBlock (/usr/local/lib/node_modules/nightwatch-html-reporter/node_modules/jade/node_modules/with/node_modules/acorn/dist/acorn.js:1938:21)
    at Parser.pp.parseStatement (/usr/local/lib/node_modules/nightwatch-html-reporter/node_modules/jade/node_modules/with/node_modules/acorn/dist/acorn.js:1689:19)
    at Parser.pp.parseFor (/usr/local/lib/node_modules/nightwatch-html-reporter/node_modules/jade/node_modules/with/node_modules/acorn/dist/acorn.js:1961:20)
    at Parser.pp.parseForStatement (/usr/local/lib/node_modules/nightwatch-html-reporter/node_modules/jade/node_modules/with/node_modules/acorn/dist/acorn.js:1770:17)
```

I have installed nightwatch by running ```npm install nightwatch -g``` which installs ```nightwatch v0.5.36```

**Comment by jls** [comment]
Looks like this started happening with Jade v1.9.2.  I'll get a fix in asap!

**Comment by jls** [comment]
Fixed in 0.3.1.  Thanks for reporting this!

**Comment by joshhornby** [comment]
Perfect :+1: Thank you
